com.unity.multiplayer.mlapi/package.json

```diff
@@ -9,5 +9,8 @@
     "unity"
   ],
   "type": "library",
-  "hideInEditor": false
-}
+  "hideInEditor": false,
+  "dependencies": {
+    "com.unity.collections": "0.14.0-preview.16"
```

Comment:
is there a non-preview one we could use?

Reply:
Not yet, this package is expected to go out of preview in Q1 of 2021. So we shall update it when it comes out of preview (We will need for UTP as well)